docs/installation/oracle.md

```diff
@@ -15,21 +15,25 @@ Support subscription to install Docker on Oracle Linux.
 
 This page instructs you to install using Docker-managed release packages and
 installation mechanisms. Using these packages ensures you get the latest release
-of Docker. If you wish to install using Oracle-managed packages, consult your
-[Oracle Linux documentation](https://oracle.com/linux).
+of Docker. If you wish to install using Oracle-managed packages, consult the
+[Oracle Linux documentation](https://docs.oracle.com/en/operating-systems/?tab=2).
 
 
 ## Prerequisites
 
 Due to current Docker limitations, Docker is only able to run only on the x86_64
 architecture. Docker requires the use of the Unbreakable Enterprise Kernel
-Release 3 (3.8.13) or higher on Oracle Linux. This kernel supports the Docker
+Release 4 (4.1.12) or higher on Oracle Linux. This kernel supports the Docker
 btrfs storage engine on both Oracle Linux 6 and 7.
 
 
 
 ## Install
 
+> *Note:* this procedure installs the non-supported upstream Docker binaries. If you wish
```

Comment:
Thanks @Djelibeybi there is a bit of confusion here on who is not supporting what. Try:

```
**Note**: This procedure installs Docker binaries. These binaries are not supported by
Oracle. To maintain Oracle Support, please follow the installation instructions
in the [Oracle Linux
documentation](http://docs.oracle.com/en/operating-systems/?tab=2).
```